infobat/irc.py

```diff
@@ -354,6 +354,12 @@ def privmsg(self, user, channel, message):
             if command_func is not None and channel_obj.is_usable(s_command[0]):
                 command_func(target, channel_obj, *s_command[1:])
 
+    def noticed(self, user, channel, message):
```

Comment:
my only comment is I'd prefer it if the code wasn't copy-pasted between privmsg and notice.

Reply:
Done